product_docs/docs/pgd/5/appusage/extensions.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Using other extensions with PGD
+navTitle: Extension usage
+---
+
+## PGD and other PostgreSQL extensions
+
+PGD is implemented as a PostgreSQL extension (see [Supported Postgres database servers
+](overview/#supported-postgres-database-servers)), taking advantage of PostgreSQL's
+expandability and flexibility to modify system behaviour at quite a low level.
```

Comment:
```suggestion
expandability and flexibility to modify low-level system behavior to enable bi-directional replication.
```

US English (sigh) spelling please. And quite is slightly equivocating/vague. Added a "why"

Reply:
Sigh... ;)

Anyway I will change `bi-directional replication` to `multi-master replication`, like wot we describe it on the very first page of the docs, as "BDR" is a bit of a misnomer.